builtin/providers/aws/resource_aws_vpc_peering_connection.go

```diff
@@ -24,10 +24,10 @@ func resourceAwsVpcPeeringConnection() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"peer_owner_id": &schema.Schema{
-				Type:        schema.TypeString,
-				Required:    true,
-				ForceNew:    true,
-				DefaultFunc: schema.EnvDefaultFunc("AWS_ACCOUNT_ID", nil),
```

Comment:
I don't think `AWS_ACCOUNT_ID` is defined anywhere, so I was under the impression that it's mainly just used to help with testing. https://www.terraform.io/docs/providers/aws/r/vpc_peering.html doesn't mention it, but rather states that both owner and peer are required, so I would imagine that unless someone was looking at source to see that `AWS_ACCOUNT_ID` was usable as a default for the variable, no one would be actually using it.